test/geometry/test_homography.py

```diff
@@ -64,14 +65,43 @@ def test_clean_points(self, batch_size, device, dtype):
         assert_allclose(
             kornia.transform_points(dst_homo_src, points_src), points_dst, rtol=1e-3, atol=1e-4)
 
-    @pytest.mark.xfail(reason='May raise checkIfNumericalAnalyticAreClose.')
     @pytest.mark.grad
-    def test_gradcheck(self, device, dtype):
-        points_src = torch.rand(1, 10, 2, device=device, dtype=torch.float64, requires_grad=True)
-        points_dst = torch.rand_like(points_src)
-        weights = torch.ones_like(points_src)[..., 0]
-        assert gradcheck(find_homography_dlt, (points_src, points_dst, weights),
-                         rtol=1e-1, atol=1e-5, raise_exception=True)
+    def test_gradcheck(self, device):
```

Comment:
`@pytest.mark.skipif(torch.__version__ > '1.7', reason="add the reason")`

Reply:
Yes!